packages/path_provider/path_provider/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## NEXT
```

Comment:
Please read the link I pointed you to again, and re-read my comments about the missing steps. This needs a version change.

Reply:
OK, i updated changelog.md and pubspace.yaml, add a new version, please review.